packages/agent/src/hl7.ts

```diff
@@ -55,6 +55,49 @@ export class AgentHl7Channel extends BaseChannel {
     }
   }
 
+  async reloadConfig(definition: AgentChannel, endpoint: Endpoint): Promise<void> {
+    const previousEndpoint = this.endpoint;
+    this.definition = definition;
+    this.endpoint = endpoint;
+
+    this.log.info('Reloading config... Evaluating if channel needs to change address...');
+
+    if (this.needToRebindToPort(previousEndpoint, endpoint)) {
```

Comment:
Only do a full restart of a channel if we need to rebind to a different interface / different port...
Otherwise we can maintain the original connections and just tweak their configurations